.github/workflows/ci-build-release-napi.yml

```diff
@@ -57,6 +65,7 @@ jobs:
           npx node-pre-gyp build --target_arch=${{ matrix.arch }}
 
       - name: Test loading Node binaries lib
+        if: matrix.arch == 'x64'
```

Comment:
Why do you skip `arm64` here? I saw that it's empty from CI: https://github.com/apache/pulsar-client-node/actions/runs/3561255235/jobs/5981995900#step:5:5

Reply:
Because GitHub's runner arch is x64, it can't run Arm64 binaries

Reply:
Get it. Thanks.